packages/orchestration/src/chain-info.js

```diff
@@ -113,6 +112,11 @@ export const registerChain = async (
       .then(() => log(`registered agoricNames chain.${name}`)),
   ];
 
+  chainInfo.assetList &&
```

Comment:
why make this an expression?
```suggestion
  if (chainInfo.assetList) {
```

Reply:
I don't remember any good reason.